hal/src/main/native/athena/AnalogGyro.cpp

```diff
@@ -169,7 +169,7 @@ void HAL_CalibrateAnalogGyro(HAL_GyroHandle handle, int32_t* status) {
 
   HAL_InitAccumulator(gyro->handle, status);
   if (*status != 0) return;
-  std::printf("Callibrating analog gyro for %d seconds.", kCalibrationSampleTime);
+  wpi::outs("Callibrating analog gyro for %d seconds.\n", kCalibrationSampleTime);
```

Comment:
That's... not how wpi::outs() works. It works like std::cout.

Reply:
oh. aha. Will fix.